docs/manual/build-api.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -e
+dir=$(dirname $0)
+
+# build API reference
+quartodoc build
+
+cd $dir/api-reference/
+if [[ -f index.qmd.qmd ]]; then
+
+    # HACK: fix generated file name
+    mv index.qmd.qmd index.qmd
```

Comment:
Is this incorrect file generated by quartodoc?

Reply:
yeah, it adds the extension twice for some reason, it's strange, probably a bug in their code.